### ISSUE TITLE
[test-runner] MaxListenersExceededWarning

### ISSUE THREAD

**Issue by Westbrook** [modernweb-dev/web#276]
The most recent version logs me a:
```
(node:30487) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 Debugger.paused listeners added. Use emitter.setMaxListeners() to increase limit
```
I wasn't getting this in `0.6.46`, but in `0.6.50` I do.

Not sure if they are related by anything but coincidence, but after that I get some:
```
Error while collecting browser logs: Evaluation failed: window.__wtr_browser_logs__ is undefined
```
Logs that I didn't get before, though not on all tests, which would seem more expected here.

**Comment by Niznikr** [comment]
I noticed the same messages during some runs though not consistently after updating from `0.6.45`.

**Comment by LarsDenBakker** [comment]
@Niznikr which browser launcher and browser are running this on?

**Comment by LarsDenBakker** [comment]
And which of the two errors reported by @Westbrook are you seeing?

**Comment by Niznikr** [comment]
@LarsDenBakker `playwright` will all three browsers. Might have to go back and rerun to see if the messages came from a certain browser.

**Comment by Niznikr** [comment]
Oh and both of them

**Comment by Niznikr** [comment]
I do have `concurrency` set to 1 currently as I get more failures when bumping it up.

**Comment by LarsDenBakker** [comment]
The same issues as listed here, or others?

**Comment by Niznikr** [comment]
Others. Just tests failing inconsistently. Might have been focus related tests similar to another issue opened by @Westbrook I believe. Just pulled in the latest releases for `@web/test-runner` and `@web/test-runner-playwright` and tried to run the tests but it failed with this error:

```
node_modules/@web/test-runner-core/node_modules/uuid/dist/esm-browser/index.js:1
export { default as v1 } from './v1.js';
^^^^^^

SyntaxError: Unexpected token 'export'
    at wrapSafe (internal/modules/cjs/loader.js:1043:16)
    at Module._compile (internal/modules/cjs/loader.js:1091:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1160:10)
    at Module.load (internal/modules/cjs/loader.js:976:32)
    at Function.Module._load (internal/modules/cjs/loader.js:884:14)
    at Module.require (internal/modules/cjs/loader.js:1016:19)
    at require (internal/modules/cjs/helpers.js:69:18)
    at Object.<anonymous> (/Users/rniznik/Documents/pharos/node_modules/@web/test-runner-core/dist/runner/createTestSessions.js:4:16)
    at Module._compile (internal/modules/cjs/loader.js:1121:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1160:10)
```

**Comment by Niznikr** [comment]
^ False alarm on that specific error. Was using an older version of `node` and forgot to reset it.

**Comment by Niznikr** [comment]
Was able to confirm I still get the `MaxListenersExceededWarning` message but not the `window.__wtr_browser_logs__ is undefined` one. I was able to set concurrency to 2 with no issues. Past that and the Firefox test failures go up.

**Comment by LarsDenBakker** [comment]
@Niznikr thanks for the info, it helps a lot. I think that's probably related to https://github.com/modernweb-dev/web/issues/238

I fixed `window.__wtr_browser_logs__ is undefined`, I found out that native warnings thrown by the browser itself don't have access to this script on the window. Not sure if it's related to timing, or just a different execution context. 

I found out that `MaxListenersExceededWarning` is a memory leak in playwright itself, I'll file an issue and maybe a PR on their side.
